tests/Oryx.BuildImage.Tests/CommandTests/OryxCommandTest.cs

```diff
@@ -89,7 +89,7 @@ public void BuildImage_Build_UsesCwd_WhenNoSourceDirGiven()
             RunAsserts(
                 () =>
                 {
-                    Assert.Contains("Error: Could not detect", result.StdErr);
+                    Assert.Contains("Required argument missing for command: 'build'.", result.StdErr);
```

Comment:
I don't think this is the error we want to be checking for -- based on the test, if there's no value provided for the `sourceDir` argument, we used the current working directory, and the error we checked for previously was that we couldn't detect a platform from the `cwd`, but now we're throwing an error saying that an argument is missing for `oryx build`, which seems to imply that we can't run `oryx build` without the `sourceDir` argument. We should double check this since it would be a breaking change if users couldn't run this command without explicitly providing a path to build.